df_gettext_toolkit/create_mod/batch.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+from typing import List
+
+import requests
+import typer
+from loguru import logger
+
+from df_gettext_toolkit.create_mod.from_template import main as from_template
+from df_gettext_toolkit.create_mod.template_from_vanilla import main as template_from_vanilla
+
+PO_URL = "https://raw.githubusercontent.com/dfint/translations-backup/main/translations/dwarf-fortress-steam"
+
+
+def fetch_po_from_git(language: str, destination_path: Path) -> None:
+    repsonse_objects = requests.get(f"{PO_URL}/objects/{language.lower()}.po")
+    with open(Path(destination_path / f"objects_{language.lower()}.po"), "w", encoding="utf-8") as file:
+        file.write(repsonse_objects.text)
+    repsonse_text_set = requests.get(f"{PO_URL}/text_set/{language.lower()}.po")
+    with open(Path(destination_path / f"text_set_{language.lower()}.po"), "w", encoding="utf-8") as file:
+        file.write(repsonse_text_set.text)
+    if repsonse_objects.status_code == 404 or repsonse_text_set.status_code == 404:
```

Comment:
There could be other statuses, so in general it's better to use [`response.raise_for_status()`](https://requests.readthedocs.io/en/latest/api/#requests.Response.raise_for_status). Also, you are writing a response text to a file after each request without status check, instead you need to add checks after each of `requests.get`.

Reply:
Then you can do
```
try:
    fetch_po_from_git(language, destination_path)
except HTTPError:
    raise Exception(f"Unable to download po file for language {language}")
```
in the `main` function

Reply:
Also, it would be useful for a use to see the actual error in addition to a custom error text